cdc/processor/memquota/mem_quota.go

```diff
@@ -51,30 +52,28 @@ type MemQuota struct {
 
 	// blockAcquireCond is used to notify the blocked acquire.
 	blockAcquireCond *sync.Cond
-	// condMu protects nothing, but sync.Cond needs a mutex.
-	condMu sync.Mutex
```

Comment:
`blockAcquireCond` shouldn't share one mutex with `tableMemory`. It will increase lock conflicts.

Reply:
`blockAcquireCond` maintains a separate lock internally that can be accessed through `blockAcquireCond.L`.